cmd/core/pdr_creation_context.go

```diff
@@ -0,0 +1,50 @@
+package core
+
+import (
+	"fmt"
+	"github.com/edgecomllc/eupf/cmd/core/service"
+	"github.com/rs/zerolog/log"
+	"github.com/wmnsk/go-pfcp/ie"
+	"net"
+)
+
+type PDRCreationContext struct {
+	Session         *Session
+	ResourceManager *service.ResourceManager
+	TEIDCache       map[uint8]uint32
+}
+
+func (pcc *PDRCreationContext) getFARID(farid uint32) uint32 {
+	return pcc.Session.GetFar(farid).GlobalId
+}
+
+func (pcc *PDRCreationContext) getQERID(qerid uint32) uint32 {
+	return pcc.Session.GetQer(qerid).GlobalId
+}
+
+func (pcc *PDRCreationContext) getFTEID(seID uint64, pdrID uint32) (uint32, error) {
+	allocatedTeid, err := pcc.ResourceManager.FTEIDM.AllocateTEID(seID, pdrID)
+	if err != nil {
+		log.Info().Msgf("[ERROR] AllocateTEID err: %v", err)
```

Comment:
Please review logs. Use specific logger level to print error message without any prefixes